packages/app/tests/components/TheFooter.spec.ts

```diff
@@ -24,8 +24,8 @@ describe("TheFooter:", () => {
       },
     });
     const links = wrapper.findAll("a");
-    expect(links[0].attributes("href")).toBe("https://docs.zksync.io/build/tooling/zksync-block-explorers");
-    expect(links[1].attributes("href")).toBe("https://zksync.io/terms");
+    expect(links[0].attributes("href")).toBe("https://docs.treasure.lol");
+    expect(links[1].attributes("href")).toBe("https://treasure.lol/terms-of-service");
     expect(links[2].attributes("href")).toBe("https://zksync.io/contact");
```

Comment:
Can delete the `expect(links[2].attributes("href")).toBe("https://zksync.io/contact");` line in the unit test because TheFooter for our explorer does not have a third link.